sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala

```diff
@@ -370,29 +370,76 @@ final class DataFrameStatFunctions private[sql](df: DataFrame) {
    * @since 1.5.0
    */
   def sampleBy[T](col: String, fractions: Map[T, Double], seed: Long): DataFrame = {
```

Comment:
@cloud-fan, WDYT about we start to deprecate String method?

Reply:
Will probably send an email after 2.4.0 since it's not going to be super urgent.